doc/playbook.rst

```diff
@@ -280,14 +280,17 @@ To inspect table permissions:
    set search_path to <namespace>;
    \dp
 
-Most tables should grant the INSERT (a), SELECT (r), UPDATE (w), and DELETE (d) `PostgreSQL privileges`_ to all service users (currently ``latiss_prompt``, ``hsc_prompt``, and ``lsstcomcamsim_prompt``).
-Sequences should have SELECT (r) and USAGE (U) instead.
+All tables should grant the INSERT (a), SELECT (r), and UPDATE (w) `PostgreSQL privileges`_ to all service users (currently ``latiss_prompt``, ``hsc_prompt``, and ``lsstcomcamsim_prompt``).
+Some tables also need DELETE (d), but we are still confirming which ones.
+
+We need SELECT (r) and USAGE (U) permissions for the sequence ``collection_seq_collection_id``, but *not* for ``dataset_calibs_*_seq_id``, ``dataset_type_seq_id``, or ``dimension_graph_key_seq_id``.
+We expect that most future sequences will only be touched by repository maintenance and not by pipeline runs or data transfers.
 
 If any tables are missing permissions, run:
```

Comment:
On Slack @ktlim  mentioned that these were manually applied
`GRANT SELECT,INSERT,UPDATE ON ALL TABLES IN SCHEMA embargo, embargo_old TO latiss_prompt, lsstcomcamsim_prompt;`

Should we just advise to grant on all tables, not selected tables?

Reply:
I'd rather keep some human judgment in the mix -- I've seen too many people interpret "run this"-type instructions as something to obey without question. That's why I said to run `\dp` and see what's there.